### ISSUE TITLE
generates HTML example

### ISSUE THREAD

**Issue by 117503445** [facebookresearch/nougat#125]
I'd like to add a feature that generates examples of HTML similar to the one in the [official website](https://facebookresearch.github.io/nougat), to make it easier to quickly evaluate results

**Comment by lukas-blecher** [comment]
I'm not sure I understand what you mean.
Do you mean something like this: https://htmlpreview.github.io/?https://github.com/Mathpix/mathpix-markdown-it/blob/master/examples/html/input-mmd-to-html.html

**Comment by 117503445** [comment]
Yes, I've tried several mmd to html converters before and none of them worked as well as the one you provided.

However I get `Undefined control sequence \textsc` and `Undefined control sequence \coloneqq`, any ideas how to fix that? Here is my file.

[hotstuff.zip](https://github.com/facebookresearch/nougat/files/12755716/hotstuff.zip)

**Comment by 117503445** [comment]
> import markdown
> 
> html = markdown.markdown(markdown_format) displayHTML(html)

Using the library [Python-Markdown](https://python-markdown.github.io/), the resulting HTML doesn't render the formulas correctly

![image](https://github.com/facebookresearch/nougat/assets/21152034/04a97376-2b82-4396-a6b1-43ee02d84e43)
